src/libmongoc/src/mongoc/mongoc-client-pool.c

```diff
@@ -47,16 +47,17 @@ struct _mongoc_client_pool_t {
    uint32_t max_pool_size;
    uint32_t size;
 #ifdef MONGOC_ENABLE_SSL
-   bool ssl_opts_set;
    mongoc_ssl_opt_t ssl_opts;
+   bool ssl_opts_set;
```

Comment:
Yes, just moving because order isn't otherwise significant and it bothers me seeing so much padding. I hadn't made any systematic investigation, this is just something I noticed while working on the code.